src/pflask.c

```diff
@@ -44,6 +44,11 @@
 #include <sys/stat.h>
 #include <sys/wait.h>
 
+#if HAVE_LIBCAP_NG
+#include <cap-ng.h>
```

Comment:
Is this include actually needed? Also, the "capabilities.h" can be left outside of the `#if`.